src/elements/common/_variables.scss

```diff
@@ -4,8 +4,9 @@
 $font: Lato, 'Helvetica Neue', Helvetica, Arial, sans-serif;
 
 $minimumWidth: 300px;
+$sidebarHeaderHeight: 60px;
 $sidebarContentWidth: 340px;
-$sidebarActivityFeedSpacingHorizontal: 25px;
+$sidebarActivityFeedSpacingHorizontal: 20px;
```

Comment:
I suspect this will introduce some inconsistency in our horizontal spacing. Should this remain 25px? Or should the new and existing styles for `.bcs-content-header` use 20px?

Reply:
Good point. I'll commit "fix" for `.bcs-content-header` to use the same variable `$sidebarActivityFeedSpacingHorizontal`